FordLabs/peoplemover
# Move currentSpace global state from redux to recoil

## Issue
Resolves N/A

## What was done
- [x] Moved currentSpace global state from redux to recoil
- [x] Updated the "Time On Product" page to fetch the current space if it wasn't already fetched so that you can go to that page directly
- [x] Fixed a bug where on the "Time On Product" page, you couldn't click on names that were on the unassigned products  

## How to test
1. Ensure functionality has stayed the same.
2. Ensure you can go to the "Time On Product" page directly vs being redirected to the peopleMover page.
3. Ensure on the "Time On Product" page, you can click on any person's name, including people that are unassigned
